tests/autopilot/autopilot_test.go

```diff
@@ -476,6 +482,61 @@ var _ = Describe(fmt.Sprintf("{%sPvcAndPoolExpand}", testSuiteName), func() {
 	})
 })
 
+var _ = Describe(fmt.Sprintf("{%sPoolResizeFailure}", testSuiteName), func() {
+	It("has to start IO workloads, create rules that resize pools based on capacity, restart autopilot and validate pools have been resized once", func() {
+		testName := strings.ToLower(fmt.Sprintf("%sPoolResizeFailure", testSuiteName))
+
+		// find the smallest pool and create a rule with size cap above that
+		var smallestPoolSize uint64
+		for _, node := range node.GetWorkerNodes() {
+			for _, p := range node.StoragePools {
+				if smallestPoolSize == 0 {
+					smallestPoolSize = p.TotalSize
+				} else {
+					if p.TotalSize < smallestPoolSize {
+						smallestPoolSize = p.TotalSize
+					}
+				}
+			}
+		}
+
+		apRules := []apapi.AutopilotRule{
+			aututils.PoolRuleByTotalSize((smallestPoolSize/units.GiB)+1, 100*16, aututils.RuleScaleTypeAddDisk, nil),
+		}
+
+		contexts := scheduleAppsWithAutopilot(testName, apRules)
+
+		err := waitForAutopilotFailedEvent(apRules, "")
+		Expect(err).NotTo(HaveOccurred())
+
+		Step("updating autopilot rules with reasonable values", func() {
```

Comment:
change `reasonable` to `correct`. The word reasonable puts things in a grey area.